calabash.gemspec

```diff
@@ -43,29 +43,33 @@ Public License.}
   spec.add_dependency 'slowhandcuke','~> 0.0', '>= 0.0.3'
   spec.add_dependency 'geocoder', '~> 1.1', '>= 1.1.8'
   spec.add_dependency 'httpclient', '~> 2.6'
-  spec.add_dependency 'awesome_print', '~> 1.6'
   spec.add_dependency 'escape', '~> 0.0', '>= 0.0.4'
-  spec.add_dependency 'CFPropertyList','~> 2.2'
   spec.add_dependency 'sim_launcher', '< 0.5', '>= 0.4.13'
   spec.add_dependency 'run_loop', '>= 1.2.2', '< 2.0'
 
   # These dependencies should match the xamarin-test-cloud dependencies.
-  spec.add_dependency 'json', '~> 1.8'
-  spec.add_dependency 'retriable', '~> 1.3', '>= 1.3.3.1'
   spec.add_dependency 'rubyzip', '~> 1.1'
 
+  # Run-loop should control the version.
```

Comment:
I don't think this has anything to do with Android vs. iOS.  It is an issue of gem dependencies.

The problem is that this gem depends on awesome-print and run-loop depends on awesome-print.

We recently had a string of problems:

* https://github.com/calabash/calabash-ios/issues/668

when we updated the awesome_print dependency in run-loop and calabash-cucumber.

This gem doesn't require any particular version of these four gems; they just have to be available.